sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/api/Azure.ResourceManager.RecoveryServicesBackup.netstandard2.0.cs

```diff
@@ -1956,7 +1648,7 @@ public partial class ILRRequestResource : Azure.ResourceManager.Models.TrackedRe
     public partial class InquiryInfo
```

Comment:
name is too general

Reply:
renamed to `WorkloadContainerInquiryInfo`